tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
It would seem everything in the `fonts.names` namespace is really only to deal with resolving names of fonts to and from file names, rather than loading the cached font table.